us.ihmc.cmap.CmapTools.appdata.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version='1.0' encoding='utf-8'?>
+<component type="desktop">
+  <!--Created with jdAppStreamEdit 7.0-->
+  <id>us.ihmc.cmap.CmapTools</id>
+  <name>CmapTools</name>
+  <summary>Allows users to to create and share concept maps.</summary>
+  <developer_name>Florida Institute for Human &amp; Machine Cognition</developer_name>
+  <launchable type="desktop-id">us.ihmc.cmap.CmapTools.desktop</launchable>
+  <metadata_license>CC0-1.0</metadata_license>
```

Comment:
no project license ?

Reply:
proprietary freeware

Reply:
it's still missing.

Reply:
Sorry, the graphically editor swallowed it.